src/ui/public/bound_to_config_obj.js

```diff
@@ -18,21 +18,17 @@ export default function BoundToConfigObjProvider($rootScope, config) {
   function BoundToConfigObj(input) {
     const self = this;
 
-    _.forOwn(input, function (val, prop) {
-      if (!_.isString(val) || val.charAt(0) !== '=') {
-        self[prop] = val;
+    _.forOwn(input, function (value, prop) {
+      if (!_.isString(value) || value.charAt(0) !== '=') {
+        self[prop] = value;
         return;
       }
 
-      const configKey = val.substr(1);
-
-      update();
-      $rootScope.$on('init:config', update);
-      $rootScope.$on('change:config.' + configKey, update);
-      function update() {
-        self[prop] = config.get(configKey);
-      }
+      const configKey = value.substr(1);
 
+      config.on(configKey, function update(value) {
+        self[prop] = value;
```

Comment:
Do these handlers get executed immediately? If not then the initial value of the property isn't getting set, right?